molecule/zabbix_server/molecule copy.yml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Is this by any chance an accidental file? Name would indicate so

Reply:
Ya it is.  I modified molecule because I haven't been able to get workflows to work locally and forgot to trash the copy.  I'll fix it and rebase tonight.  Any idea why sanity would just now start throwing errors since I didn't touch any of the plugins.

Reply:
since we are testing against `develop` branch from ansible/ansible as well, we are subject to such surprises when new standards are merged in. This sometimes happens even before it gets announced at ansible-collections/new-for-maintainers, not much to do.

So it was just a bad luck